src/components/form.Select/form.Select.tsx

```diff
@@ -33,36 +34,46 @@ const FormSelect = ({
     if (!errors[name]) {
       alert(`You have selected ${data[name]}`);
     } else {
+      console.log(errors);
+
       alert(`Please select ${title.toLowerCase()}.`);
     }
   };
 
   return (
-    <Form
-      onSubmit={handleSubmit(onSubmit)}
-      className=" text-black max-w-pageContent w-86 h-auto flex flex-col	m-0 px-4	"
-    >
-      <label htmlFor={name as string} className=" font-text">
-        {title}
-      </label>{" "}
-      <select
-        id={name as string}
-        {...register(name as string, { required: true })}
-        defaultValue=""
-        className="m-0 box-border  border shadow-sm rounded-lg grow-0 h-14 p-2  w-80  font-text"
-      >
-        <option className="opacity-50	" value="" disabled hidden>
-          {placeholderValue}
-        </option>
-
-        {options.map((option) => (
-          <option key={option.value} value={option.value}>
-            {option.label}
+    <form onSubmit={handleSubmit(onSubmit)}>
```

Comment:
should be wrapped in the from component wrap in the form.stories not inside the components